src/palo_alto_firewall_analyzer/pan_config.py

```diff
@@ -40,10 +41,9 @@ def get_device_groups(self):
 
     @functools.lru_cache(maxsize=None)
     def get_device_groups_hierarchy(self):
-        xpath = "./config/readonly/devices/entry[@name='localhost.localdomain']/device-group/entry"
-
+        xpath = "./config/readonly/devices/entry[@name='localhost.localdomain']/device-group/entry"        
         device_group_hierarchy_children = collections.defaultdict(list)
-        device_group_hierarchy_parent = {}
+        device_group_hierarchy_parent = {}        
```

Comment:
Could you please remove this extra whitespace?

Reply:
fixed